app/marketing/mails.py

```diff
@@ -884,6 +884,147 @@ def funder_payout_reminder(to_email, bounty, github_username, live):
         return html
 
 
+def grant_match_distribution_kyc(match):
+    to_email = match.grant.admin_profile.email
+    cc_emails = [profile.email for profile in match.grant.team_members.all()]
+    from_email = 'kyc@gitcoin.co'
+    cur_language = translation.get_language()
+    rounded_amount = round(match.amount, 2)
+    token_name = f"CLR{match.round_number}"
+    try:
+        setup_lang(to_email)
+        subject = f"💰 (ACTION REQUIRED) Grants Round {match.round_number} Match Distribution: {rounded_amount} DAI"
+        body = f"""
+<pre>
+Hello @{match.grant.admin_profile.handle},
+
+This email is in regards to your Gitcoin Grants Round {match.round_number} payout of {rounded_amount} DAI for https://gitcoin.co{match.grant.get_absolute_url()}.
+
+We are required by law to collect the following information from you in order to administer your payout.  Please respond to this email with the following information.
+
+Full Name:
+Physical Address:
+(Only if you’re a US Citizen) Social Security Number:
+Proof of physical address (utility bill, or bank statement)
+Proof of identity (government issued ID)
+
+Thanks,
+Gitcoin Grants KYC Team
+</pre>
+
+        """
+        send_mail(
+            from_email,
+            to_email,
+            subject,
+            '',
+            body,
+            from_name=_("Gitcoin Grants"),
+            cc_emails=cc_emails,
+            categories=['admin', func_name()],
+        )
+    finally:
+        translation.activate(cur_language)
+
+
+def grant_match_distribution_test_txn(match):
+    to_email = match.grant.admin_profile.email
+    cc_emails = [profile.email for profile in match.grant.team_members.all()]
+    from_email = 'kyc@gitcoin.co'
+    cur_language = translation.get_language()
+    rounded_amount = round(match.amount, 2)
+    token_name = f"CLR{match.round_number}"
+    coupon = "Pick up ONE item of Gitcoin Schwag at http://store.gitcoin.co/ at 25% off with coupon code {settings.GRANTS_COUPON_25_OFF}"
+    if match.amount > 100:
+        coupon = "Pick up ONE item of Gitcoin Schwag at http://store.gitcoin.co/ at 50% off with coupon code {settings.GRANTS_COUPON_50_OFF}"
+    if match.amount > 1000:
+        coupon = "Pick up ONE item of Gitcoin Schwag at http://store.gitcoin.co/ at 100% off with coupon code {settings.GRANTS_COUPON_100_OFF}"
+    # NOTE: WE WILL SCREEN AND APPROVE ANY ORDERS TO THE STORE, SO IF YOURE A CLEVER BISCOUT
```

Comment:
LOL - i think we actually dont have to worry about this, since they are set as ENV vars :) you should make the default worth 10% off or something though and add a message here saying since you were reading our codebase you get a discount!

Reply:
maybe ill make a 100% off code with a limit of 1 or 2 purchases and sneak it in here as an easter egg.